microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -594,6 +594,439 @@
+                    .attr("id", "boxplot-modal")
+                    .attr("class", "boxplot-modal");
+
+                modal.append("div")
+                    .attr("class", "boxplot-overlay")
+                    .on("click", closeModal);
+
+                const modalContent = modal.append("div")
+                    .attr("class", "boxplot-content");
+
+                modalContent.append("span")
+                    .attr("class", "close-button")
+                    .html("&times;")
+                    .on("click", closeModal);
+
+                modalContent.append("h4")
+                    .style("text-align", "left")
+                    .style("color", "#222")
+                    .text("How to read a boxplot (horizontal)");
+
+                modalContent.append("svg")
+                    .attr("id", "tooltip-boxplot")
+                    .attr("width", 605)
+                    .attr("height", 223);
+
+                
+                const infoContainer = modalContent.append("div")
+                    .attr("class", "info-paragraph-container")
+                    .style("display", "flex")
+                    .style("align-items", "left")
+                    .style("gap", "8px")
+                    .style("margin-bottom", "15px");
+
+                infoContainer.append("div")
+                    .attr("class", "info-circle")
+                    .style("width", "20px")
+                    .style("height", "20px")
+                    .style("background-color", "#006A96")
+                    .style("border-radius", "50%")
+                    .style("display", "flex")
+                    .style("justify-content", "center")
+                    .style("align-items", "center")
+                    .style("font-size", "14px")
+                    .style("color", "white")
+                    .style("font-weight", "bold")
+                    .text("i");
+
+                infoContainer.append("p")
+                    .style("margin", "0")
+                    .style("font-size", "12px")
+                    .style("color", "#006A96")
+                    .style("line-height", "1.5")
+                    .style("align-items", "center")
+                    .style("text-align", "left") 
+                    .text("To clearly show the abundance of microbes in your sample, the scale on the data plots may vary across figures to focus on the relevant range for each comparison.");
+
+                modalContent.append("p")
+                    .style("text-align", "left") 
+                    .text("Example: Microbe A is a dominant microbe, exceeding 50% of the abundance in most samples. Microbe B is typically much less abundant at less than 10%. In the case of microbe B, the plot will be much more readable if the scale only shows up to 15% rather than 100%. ");
+
+                drawExampleBoxplot();
+            }
+        });
+
+        function drawExampleBoxplot() {
```

Comment:
There's a lot of repeated code between this and the code that draws the actual boxplots. Can we create a helper function that just handles the actual drawing and can be fed information to draw either real or example boxplots?